cloudferrylib/os/compute/server_groups.py

```diff
@@ -45,16 +50,18 @@
                      "deleted) VALUES({0}, '{1}', 0)")
 
 
-class Handler(Resource):
+class ServerGroupsHandler(compute.Compute):
     """
     Handler for Nova Server/Instance Groups on specified cloud.
     Allows for collection, creation and duplicate detection.
     """
 
     def __init__(self, cloud):
-        super(Handler, self).__init__()
+        super(ServerGroupsHandler, self).__init__()
         self.cloud = cloud
-        self.compute = self.cloud.resources[utl.COMPUTE_RESOURCE]
+        self.compute = self.cloud.resources[utils.COMPUTE_RESOURCE]
+        self.identity = self.cloud.resources[utils.IDENTITY_RESOURCE]
+        self.config = copy.deepcopy(self.identity.config)
```

Comment:
Deep copy doesn't seem to be needed, unless `self.config` gets modified somewhere (which I didn't spot)